### ISSUE TITLE
The `update.ts` script fails when the `deno.jsonc` files uses comments

### ISSUE THREAD

**Issue by frou** [denoland/fresh#1576]
Files with the `.jsonc` extension allow comments, but the update script can't cope with that because it tries to parse `deno.jsonc` with a normal JSON parser:

```
error: Uncaught (in promise) SyntaxError: Expected double-quoted property name in JSON at position 182
let denoJson = JSON.parse(denoJsonText);
                    ^
    at JSON.parse (<anonymous>)
    at https://deno.land/x/fresh@1.3.1/update.ts:52:21
    at eventLoopTick (ext:core/01_core.js:183:11)
```

Reading, modifying and writing back out the file while preserving comments is probably tricky. But I guess not preserving comments would be preferable to failing.

**Comment by deer** [comment]
Thanks for logging this. Since I didn't really fix this the first time around in #1349, I'll try again here. This time the test will actually make sure there's a comment involved.

**Comment by frou** [comment]
Thank you sir
